### DISCUSSION TITLE
Version v0.2.0 and later Dashboard Menu Errors

### DISCUSSION THREAD

**Discussion by rallyrabbit** [LunarVim/LunarVim#1887] [Q&A] [answered]
I'm getting an odd issue, has anyone seen:
Error detected while processing function <SNR>59_call_line_function:
line    6:
E492: Not an editor command: Telescope find_files

This is happenning off the main dashboard menu.  Seems to be intermittent after change #560

I think there may be some startup errors, but they scroll off the screen

**Comment by mandaputtra** [comment]
Try update the package

```
:PackerUpdate
:PackerCompile
```

There are some change on the telescope package

**Reply by rallyrabbit** [reply]
Hi,

I did that but thank you for the suggestion.  I also blew away the nvim data directory to allow everything to load fresh.  Which also failed.  One note, I am gradually trying to get this to work in windows by taking change by change to see where things break.  The only thing I can say is something between change 559 and 564 something broke with dashboard and telescope, but the odd thing is, I see no breaking changes in the code itself.

**Reply by mandaputtra** [reply]
Hmm, I always use latest rolling branch so I'm not encountered by this issue.

It is actually hard to debug an editor ~ this is also why I use lunarvim.

Maybe for now just use the latest rolling branch? I don't see any errors on rolling branch for now so...

**Comment by rallyrabbit** [comment]
Unfortunately the rolling latest is even worse, that's why I stated with a known state and started rolling up to make a list of dependencies.  This one is one I cannot figure out.

Is there any way to dump the contents of the exceptions and errors that show up in the bottom of the VIM area (status line and below)

**Reply by chase** [reply]
> Is there any way to dump the contents of the exceptions and errors that show up in the bottom of the VIM area (status line and below)

`:messages`

**Comment by rallyrabbit** [comment]
Also, there's something about the changes for the lazy loading that seemed to make this happen, that's at least what I've traced so far

**Comment by abzcoding** [comment, marked as answer]
we've replaced dashboard with alpha.nvim
